package.json

```
@@ -38,7 +38,7 @@
     "yarn": "^1.22.17"
   },
   "scripts": {
-    "start": "HTTPS=true react-scripts start",
+    "start": "react-scripts start",
```

HTTPS=true  
```suggestion
    "start": "HTTPS=true react-scripts start",
```
이 부분은 제거하겠습니다.